helm-chart/README.md

```diff
@@ -78,7 +78,10 @@ helm install --name my-release -f myvalues.yaml .
 | `service.port`                    | Port the connector pods will accept connections on | `9201`                      |
 | `service.loadBalancer.enabled`    | If enabled will create an LB for the connector, ClusterIP otherwise | `true`     |
 | `service.loadBalancer.annotations`| Annotations to set to the LB service        | `service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "4000"` |
-| `dropChunk.schedule`              | The schedule with which the drop-chunk Job runs | `0,30 * * * *`                 |
+| `maintenance.schedule`            | The schedule with which the Job, that deletes data outside the retention period, runs | `0,30 * * * *` |
```

Comment:
Can you add a note to the draft release(https://github.com/timescale/promscale/releases/tag/untagged-c1fb4c9335e96b50884f) about the change in parameter name?